src/main/java/seedu/notus/data/notebook/Notebook.java

```diff
@@ -247,6 +247,16 @@ public boolean getNote(String noteTitle) {
         return notes.stream().anyMatch(note -> note.getTitle().equalsIgnoreCase(noteTitle));
     }
 
+    /**
+     * Returns true if there is a matching note in the archived notes based on the note title parameter.
+     *
+     * @param noteTitle String containing the title of the note.
+     * @return if the note exist.
+     */
+    public boolean getArchiveNote(String noteTitle) {
+        return archivedNotes.stream().anyMatch(note -> note.getTitle().equalsIgnoreCase(noteTitle));
+    }
+
```

Comment:
What is this used for?

Reply:
Currently adding of note doesn't check if the title match any archive notes. So I added this. Refer to my changes in AddNoteCommand

Reply:
Yea I realized but I forgot to remove this comment! But yeap thanks!